src/openapi_parser/loose_types.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class LooseContentType:
+    value: str
+
+
+@dataclass
+class LooseIntegerFormat:
+    value: str
+
+
+@dataclass
+class LooseNumberFormat:
+    value: str
+
+
+@dataclass
+class LooseStringFormat:
+    value: str
```

Comment:
I opted for separate classes for the sake of being explicit, but this could also very well be simplified to:

```py
@dataclass
class LooseEnum:
    value: str


LooseContenType = LooseEnum
LooseIntegerFormat = LooseEnum
LooseNumberFormat = LooseEnum
LooseStringFormat = LooseEnum
```